tensorboard/components/tf_backend/test/backendTests.ts

```diff
@@ -105,6 +105,20 @@ describe('backend tests', () => {
               '/data/plugin/scalars/scalar');
         });
 
+        it('ignores leading slash in dataDir', function() {
```

Comment:
Can we standardize the naming on "pathPrefix" as you used for createProdPath(), rather than "dataDir"?  It's confusing that it's inconsistent between parts of the code and code vs tests, and "pathPrefix" both is clearer and matches the flag naming.

Nothing except these two files seems to reference dataDir, so I think just a find-replace would suffice.